### DISCUSSION TITLE
Many genes with low pct1 and pct2 in DE analysis

### DISCUSSION THREAD

**Discussion by ruggerobelluomo** [satijalab/seurat#8439] [General]
Hi everyone, I have a question about DE analysis with single-nuclei rna seq data generated with 10x protocols. I find a lot of genes that are expressed in low pct (0.07 or so) with high FC and low pval. I was wondering, since 10x relies on sequencing a lot of cells but only a few genes per cell in order to capture most genes, does this affect the fact that I see some genes only in a low percentage of cell pooulation? Would it be different if I used for instance smartseq which sequences much deeper but in fewer cells?  

Thanks!

**Comment by denvercal1234GitHub** [comment]
pct seems to be proportion - not percentages?
